fed/mosaic-cell/src/main/resources/CNetworkScheme.json

```diff
@@ -1,180 +1,304 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "CNetworkScheme",
+    "description": "Schema describing the JSON file structure for the network configuration. It is is used to define the properties of the global network. The value should be given in gigabit if defined as number. Alternatively this can be defined as a string to specify the unit of measurement (e.g. '3 MB').",
```

Comment:
> ~~The value should be given in gigabit if defined as number. Alternatively this can be defined as a string to specify the unit of measurement (e.g. '3 MB').~~